README.md

```diff
@@ -184,17 +185,72 @@ test('should render mock graphql responses', async () => {
 })
 ```
 
+### End-to-end flows
+
+We offer hooks to make writing end-to-end tests easier.
+
+Extended commands
+
+- clickById
+- getById
+- typeById
+
+Custom methods
+
+- goToSearchPage
+- goToProductPageByShelf
+- checkText
+- checkIfElementExists
+- openCart
+- closeCart
+- clearCart
+- fillAndCheckShippingSimulator
+- scrollToId
+
+Example:
+
+```js
+import { openCart } from "@vtex/test-tools/cypress"
+
+describe('Searchbar', () => {
+  before(() => {
+    cy.intercept(/operationName=ProductsSuggestionsQuery/, {
+      fixture: 'product-suggestions-query.json',
+    }).as('searchSuggestionsLoad')
+  })
+
+  it('finds a very specific product using the search bar', () => {
+    cy.visit('/')
+
+    cy.getById('searchBarInput').eq(0).type('Product')
+
+    cy.getById('searchSuggestionsItem').contains('Product').click()
+
+    cy.clickById('addToCart')
+
+    openCart()
+
+    ...
+  })
+})
+```
+
+You can check more details of these hooks in the links below:
+
+- [`cypress/support/commands`](https://github.com/vtex/test-tools/tree/master/cypress/support/commands.ts)
+- [`cypress/utils`](https://github.com/vtex/test-tools/tree/master/cypress/utils)
+
 ## Examples
 
 These are some common use cases that might be helpful to see how it's done:
 
 ### [Debugging tests in VS Code](./examples/vscode/)
 
-### [Testing a React component](https://github.com/klzns/test-repo/blob/master/react/JsComponent.test.js)
-
 ### [Testing a React Hook](./examples/hooks/)
 
-### [TypeScript](./examples/typescript/)
+### [Testing a React component](https://github.com/klzns/test-repo/blob/master/react/JsComponent.test.js)
+
+### [Testing a React component using Typescript](./examples/typescript/)
```

Comment:
can you convert all of these to a list? it kinda bugs me that these are all headings 😅